src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -97,7 +135,7 @@ export default {
 
       this.context = context
       this.runner = new FlowRunner(context)
-      await this.goNext()
+      await this.goNext(0)
```

Comment:
Why this param `0` ? the `goNext()` method doesn't accept any params

Reply:
Missed removing this earlier. Fixed it now!